spec/activerecord-multi-tenant/query_rewriter_spec.rb

```diff
@@ -73,7 +73,7 @@
         end
       end.to change { Project.count }.from(3).to(1)
       query_index = 0
-      @queries.each_with_index do |actual_query, index|
+      @queries.each_with_index do |actual_query, _index|
```

Comment:
Why not use ‘each‘ instead of ‘each_with_index‘?

Reply:
Unused index is removed now.